src/seaport.ts

```diff
@@ -867,12 +869,38 @@ export class OpenSeaPort {
     const { buy, sell } = assignOrdersToSides(order, matchingOrder)
 
     const metadata = this._getMetadata(order, referrerAddress)
-    const transactionHash = await this._atomicMatch({ buy, sell, accountAddress, metadata })
+
+    let transactionHash: string = '';
+
+    if (isFeeWrapperStaticTarget({buy, sell})) {
```

Comment:
Can we add comments for why we're doing this and context?

Reply:
Do we need to check fee recipient is fee wrapper?

Reply:
hmm, we could but that should be validated already on our backend (i.e. a user should never be able to create an order with the fee recipient not being the wrapper)